org.eclipse.xtext/src/org/eclipse/xtext/naming/CopyQualifiedNameService.java

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+ * Copyright (c) 2018 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.naming;
+
+import org.eclipse.emf.ecore.EObject;
+
+/**
+ * Replacement for org.eclipse.xtext.ui.editor.copyqualifiedname.CopyQualifiedNameService
+ * 
+ * @author Anton Kosyakov - Initial contribution and API
+ * @author Arne Deutsch - Moved to new package
+ * @since 2.14
+ */
+public interface CopyQualifiedNameService {
```

Comment:
@szarnekow when do we start interfaces with `I` and when don't we.
how was this decided in the past?

Reply:
Please add prefix "I" as it is most common

Reply:
There is no strict rule throughout the code base, but heuristically it's better use the I prefix.